vanilla-framework/vanilla-docs-theme
# Image border

## Done

Implemented `.u-image-frame`.

## QA

- Pull branch
- Run `./run serve`
- Goto http://0.0.0.0:8103/vanilla-docs-theme/examples/utilities/image-frame/image-frame/
- All images should have a grey border as per https://app.zenhub.com/workspace/o/vanilla-framework/vanilla-docs-theme/issues/12#issuecomment-272431040

## Details

Fixes https://github.com/vanilla-framework/vanilla-docs-theme/issues/12

## Screenshots

<img width="386" alt="screen shot 2017-07-21 at 10 49 23" src="https://user-images.githubusercontent.com/479384/28458799-ee87885c-6e02-11e7-8095-cdf70e92b590.png">
